### PR TITLE
automatically build an admin/admin superuser in the docker workflow

### PR DESCRIPTION
**Pull request by Pomax**
Closes https://github.com/mozilla/foundation.mozilla.org/issues/4362 by updating the `docker_create_super_user` task to run the user creation as a direct python one-liner, rather than using the TTY `manage createsuperuser` command.

This means there will now always be an `admin`/`admin` superuser when running `inv docker-new-env` and `inv docker-new-db` without needing to specify your own.

### CODE REVIEW

File: tasks.py [hunk 1]
```diff
@@ -189,18 +189,8 @@ def docker_l10n_block_inventory(ctx):
...
-              "To create an admin user:\n"
-              "docker-compose run --rm backend pipenv run python network-api/manage.py createsuperuser\n")
-    else:
-        print("* Creating superuser.")
-        ctx.run(
-            "docker-compose run --rm backend pipenv run python network-api/manage.py createsuperuser",
-            pty=True
-        )
-
+    docker_manage(ctx, "shell -c \"from django.contrib.auth.models import User; User.objects.create_superuser('admin', 'admin@example.com', 'admin')\"")
```
Comment (cadecairos) [inline tasks.py]: Flake8 sez: "line 2 loooong"

File: tasks.py [hunk 2]
```diff
@@ -189,18 +189,8 @@ def docker_l10n_block_inventory(ctx):
...
+    print("\nCreated superuser `admin` with password `admin`.")
```
Comment (cadecairos) [inline tasks.py]: need an additional line break here.

File: tasks.py (hunk 1 above)
Comment (Pomax) [reply tasks.py]: damnit flake8

File: tasks.py (hunk 2 above)
Comment (Pomax) [reply tasks.py]: Where do you want it, before or after?

**Pull request merged**